tests/unit/views/HelloWorld.test.js

```diff
@@ -54,8 +51,7 @@ describe('HelloWorld.vue', () => {
 
     test('Render default contents', async () => {
       const wrapper = await setupWrapper();
-
-      expect(wrapper)
+      expect(wrapper.html())
```

Comment:
Do not pass in `wrapper.html()` unless you are having a specific snapshot error that requires it. Giving the serializer the entire Vue VDOM object gives it more data to be able to create better snapshots.

```suggestion

      expect(wrapper)
```